cli/src/wallet_cli/completer.rs

```diff
@@ -7,52 +7,9 @@ use colored::Colorize;
 use rustyline::{
     completion::Completer, highlight::Highlighter, hint::HistoryHinter, Completer, Context, Helper, Hinter, Validator,
 };
+use strum::VariantNames;
 
-const WALLET_COMMANDS: &[&str] = &[
-    "accounts",
-    "address",
-    "allot-mana",
-    "balance",
-    "burn-native-token",
-    "burn-nft",
-    "claim",
-    "claimable-outputs",
-    "clear",
-    "congestion",
-    "consolidate",
-    "create-account-output",
-    "create-native-token",
-    "destroy-account",
-    "destroy-foundry",
-    "exit",
-    "faucet",
-    "implicit-account-creation-address",
-    "implicit-account-transition",
-    "implicit-accounts",
-    "melt-native-token",
-    "mint-native-token",
-    "mint-nft",
-    "node-info",
-    "output",
-    "outputs",
-    "send",
-    "send-native-token",
-    "send-nft",
-    "sync",
-    "transaction",
-    "transactions",
-    "tx",
-    "txs",
```

Comment:
This used to contain shortcuts though, but I'm not too bothered removing them

Reply:
Could be added to the iterator part if we want it